lib/progressive/projects_helper_patch.rb

```diff
@@ -24,16 +24,21 @@ def render_project_hierarchy_with_progress_bars(projects)
       def render_project_progress_bars(project)
         project.extend(Progressive::ProjectDecorator)
         s = ''
-        if project.issues.open.any?
+        # If the project has or does not have issues the progress bar is shown
+        # If project.issues.count > 0 the progress bar is shown only if you have issues open or closed
+        # if project.issues.open.any? the progress bar is shown only if you have open issues
+        if project.issues.count >= 0
```

Comment:
@RuslanRadko please create a separate PR with this change. It looks quite relevant.